docs/guides/prefect-deploy.md

```diff
@@ -267,6 +267,15 @@ Use the tabs below to explore both deployment creation options.
 
     See all the optional keyword arguments for the DeploymentImage class [here](https://docker-py.readthedocs.io/en/stable/images.html#docker.models.images.ImageCollection.build). 
 
+    !!! tip "Default Docker registry URL"
+        You can use the `PREFECT_DEFAULT_DOCKER_BUILD_NAMESPACE` setting to set a default Docker namespace to append to all images you build with `.deploy`. This is great if you use a private registry to store your images.
+
+        To set a default Docker namespace for your current profile run:
+
+        ```bash
+        prefect config set PREFECT_DEFAULT_DOCKER_BUILD_NAMESPACE=<docker-registry-url>/<organization-or-username>
+        ```
+
```

Comment:
what do you think about adding a very brief concrete example of what a resulting `repo/image:tag` would end up as for a given `PREFECT_DEFAULT_DOCKER_BUILD_NAMESPACE` setting? e.g. dockerhub, since I'm not totally clear upon reading the above